pom.xml

```diff
@@ -140,6 +140,31 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.sundr</groupId>
+                <artifactId>sundr-maven-plugin</artifactId>
+                <version>${sundrio.version}</version>
```

Comment:
I think you should add `<inherited>false</inherited>` false to it.

Reply:
Done